yarn.lock

```diff
@@ -24803,6 +24803,23 @@ workspace-tools@^0.10.1:
     multimatch "^4.0.0"
     read-yaml-file "^2.0.0"
 
+workspace-tools@^0.10.2:
```

Comment:
Can you update our version of workspace-tools in scripts to latest as well to get rid of the duplicate?

Reply:
done